crates/ra_proc_macro_srv/src/proc_macro/mod.rs

```diff
@@ -201,54 +201,54 @@ impl Span {
     }
 
     /// The original source file into which this span points.
-    pub fn source_file(&self) -> SourceFile {
+    pub fn source_file(self) -> SourceFile {
```

Comment:
This part of code related to libproc-macro abi, maybe leave it as is ?

Reply:
I can drop changes in `ra_proc_macro_srv` if that's what you prefer.

Reply:
Yeah.

Basically all files inside `ra_rpoc_macro_srv/src/proc_macro/` are ABI related.